packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -61,17 +61,32 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
         return details?.login;
     }, [details, dxtradePlatform, mt5Platform, platform]);
 
-    const showNoNewPositionsMessage = useMemo(() => {
-        return (
-            !activeWalletData?.is_virtual &&
-            details?.landing_company_short === 'svg' &&
-            ['synthetic', 'financial'].includes(marketType ?? '') && (
-                <InlineMessage type='warning' variant='outlined'>
-                    No new positions
-                </InlineMessage>
-            )
-        );
-    }, [activeWalletData?.is_virtual, details?.landing_company_short, marketType]);
+    const showMigrationMessage = useMemo(() => {
```

Comment:
minor - naming, 
"showMigrationMessage" suggests that it is a boolean/flag, given its .jsx, probably it should be just "migrationMessage"